### DISCUSSION TITLE
linux 已经手动安装过 v2 版本的如何升级 v3

### DISCUSSION THREAD

**Discussion by Sfancy** [AlistGo/alist#2886] [Q&A]
**Linux** 服务器通过**手动**的形式安装的 **V2 版本**，路径为：`/root/alist`

文件夹内有三个文件，分别为：
1. `alist-linux-amd64`
2. `alist-linux-amd64.tar.gz`
3. `data`

添加了守护进程， `systemctl status alist` 可以通过 命令查看运行状态。

```
root@localhost:~/alist/data# systemctl status alist
● alist.service - alist
   Loaded: loaded (/etc/systemd/system/alist.service; disabled; vendor preset: enabled)
   Active: active (running) since Thu 2022-09-01 00:35:45 EDT; 4 months 2 days ago
 Main PID: 10444 (alist-linux-amd)
    Tasks: 5 (limit: 4915)
   CGroup: /system.slice/alist.service
           └─10444 /root/alist/alist-linux-amd64 -conf /root/alist/data/config.json
```

我有以下问题：
1. V3 版本与 V2 版本不兼容，如何卸载手动安装的 V2 版本的程序？
2. 守护进程 `/etc/systemd/system/alist.service` 里的文件是否需要修改或删除？
3. V2 的 `data` 中已经添加过的网盘内容能否兼容 V3 版本，是否需要备份？

谢谢🙏～

**Comment by NovaShen0724** [comment]
1直接删除即可
2你对照下文件目录和二进制文件名相同就不用改
3需要
